features/app-file-handling.yml

```
@@ -0,0 +1,11 @@
+name: File handling
```

Now that we have pluralized all the others, let's make this one consistent.

```suggestion
name: File handlers
```